### PR TITLE
Pull in text to summarize principles

### PR DESCRIPTION
**Pull request by tfpauly**
This is what I’ve written based on our discussion the week before IETF 110, as an attempt to write down the high level guidance and principles we could provide (from my perspective). I’m not sure the best way that this meshes with the current structure, but wanted to share to start a discussion.

### CODE REVIEW

File: draft-iab-path-signals-collaboration.md [hunk 1]
```diff
@@ -145,6 +145,62 @@ useful models to apply.
...
+of a case where there is an explicit authentication and negotiation with a
+network path element that’s used to optimize behavior or gain access to
+specific resources. 
+
+The goal of improving privacy and trust on the Internet does not necessarily
+need to remove the ability for network elements to perform beneficial
+functions. We should instead improve the way that these functions are
+achieved. Our goals should be:
+- To ensure that information is distributed intentionally, not accidentally; and,
+- To gate the distribution of information on the consent of the relevant parties
```
[the remark this answers is not in the document]
Comment (tfpauly) [reply draft-iab-path-signals-collaboration.md:176]: I'm a bit hesitant to add the question about risk of misuse. I think once we start guessing at how data could be used, we open up holes to attacks we don't expect. I think it's better to just have trust for any information in general. 

Note that some information, like timing/RTT/loss may arguably belong more to the path, than the endpoints.

[the remark this answers is not in the document]
Comment (mirjak) [reply draft-iab-path-signals-collaboration.md:176]: I was think about the spin bit case where we concluded that the data provided doesn't really add an additional risk (given you often roughly now the location based on IP address and the spin bit is to coarse grained) and as such it is okay to "broadcast" without trust. So I think you need to first understand "how much trust" is needed before you can decide how to share any information.

**Comment by jariarkko** [comment]
Looks good to me. May need some references to the path signals rfc etc.

File: draft-iab-path-signals-collaboration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jariarkko) [reply draft-iab-path-signals-collaboration.md:176]: FWIW, I agree with Mirja's point about the analysis and the Spin bit example is one where this actually had relevance.

**Review by jariarkko — APPROVED** [review]

File: draft-iab-path-signals-collaboration.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jariarkko) [reply draft-iab-path-signals-collaboration.md:176]: I made some edits in the most recent version.

**Pull request merged**